src/scripts/vendor/polyfill.js

```diff
@@ -0,0 +1,1297 @@
+      return this._stylesheets
+        && this._stylesheets.length
+        && this._stylesheets[0].text },
+    function() {
+      var i = 0
+        , stylesheet
+      while (stylesheet = this._stylesheets[i++]) {
+        stylesheet.rules = StyleManager.parse(stylesheet.text, stylesheet.url)
+      }
+    }
+  )
+}
+
+Polyfill.prototype._filterCSSByKeywords = function() {
+  this._defer(
+    function() {
+      return this._stylesheets
+        && this._stylesheets.length
+        && this._stylesheets[0].rules
+    },
+    function() {
+      var stylesheet
+        , media
+        , rules = []
+        , i = 0
+      while (stylesheet = this._stylesheets[i++]) {
+        media = stylesheet.media
+        // Treat stylesheets with a media attribute as being contained inside
+        // a single @media block, but ignore `all` and `screen` media values
+        // since they're basically meaningless in this context
+        if (media && media != "all" && media != "screen") {
+          rules.push({rules: stylesheet.rules, media: stylesheet.media})
+        } else {
+          rules = rules.concat(stylesheet.rules)
+        }
+      }
+      this._filteredRules = StyleManager.filter(rules, this._options.keywords)
+    }
+  )
+}
+
+Polyfill.prototype._buildMediaQueryMap = function() {
+  this._defer(
+    function() { return this._filteredRules },
+    function() {
+      var i = 0
+        , media
+        , rule
+      this._mediaQueryMap = {}
+      while (rule = this._filteredRules[i++]) {
+        if (rule.media) {
+          media = rule.media.join(" and ")
+          this._mediaQueryMap[media] = MediaManager.matchMedia(media)
+        }
+      }
+    }
+  )
+}
+
+Polyfill.prototype._reportInitialMatches = function() {
+  this._defer(
+    function() {
+      return this._filteredRules && this._doMatched
+    },
+    function() {
+      this._doMatched(this.getCurrentMatches())
+    }
+  )
+}
+
+Polyfill.prototype._addMediaListeners = function() {
+  this._defer(
+    function() {
+      return this._filteredRules
+        && this._doMatched
+        && this._undoUnmatched
+    },
+    function() {
+      EventManager.addListeners(
+        this,
+        function(query, isMatch) {
+          var i = 0
+            , rule
+            , matches = []
+            , unmatches = []
+          while (rule = this._filteredRules[i++]) {
+            if (rule.media && rule.media.join(" and ") == query) {
+              (isMatch ? matches : unmatches).push(rule)
+            }
+          }
+          matches.length && this._doMatched(new Ruleset(matches))
+          unmatches.length && this._undoUnmatched(new Ruleset(unmatches))
+        }
+      )
+    }
+  )
+}
+
+Polyfill.modules = {
+  DownloadManager: DownloadManager,
+  StyleManager: StyleManager,
+  MediaManager: MediaManager,
+  EventManager: EventManager
+}
+Polyfill.constructors = {
+  Ruleset: Ruleset,
+  Rule: Rule
+}
+
+window.Polyfill = Polyfill
+
+}(window, document));
```

Comment:
It gets combined into `cyclops.js` (see https://github.com/CenturyLinkCloud/Cyclops/pull/70/commits/187bf077c755f70a8c1ce2ee3939b2a89f41c005#diff-97599b685e6282e123c02b0f1f23a51dR91).